### ISSUE TITLE
Can't type 'q'

### ISSUE THREAD

**Issue by j123123** [pwdhash/pwdhash-webextension#13]
For some unknown reason I can't type 'q' letter in any textbox when pwdhash extention enabled. It affect only lowercase 'q'

**Comment by j123123** [comment]
#10 - similar issue. But I even can't type 'q' in this comment in github if extension enabled. This is not something gmail-specific. It doesn't work everywhere.
Tested in Firefox version 66.0.2 (64-bit) (Ubuntu 18.04.2 LTS build)

**Comment by Tigon3rd** [comment]
Yep, can confirm same issue. Although small 'q' works for me in the address bar and search bar of firefox, it doesn't work on sites/pages loaded into the browser.

Tested in Firefox Quantum 66.0.3 (64-bit), Windows 7 Ultimate 64-bit (Version 6.1, Build 7601 Service Pack 1)

**Comment by thennen** [comment]
I have the same issue! (66.0.2)

**Comment by radiosilence** [comment]
I have this issue in FF 70 MacOS
